circle.yml

```diff
@@ -24,7 +24,7 @@ general:
 
 deployment:
   release:
-    tag: /(([a-z]+)-)?([0-9]+)\.([0-9]+)\.([0-9]+)/
+    tag: /(([a-z]+)-)*([0-9]+)\.([0-9]+)\.([0-9]+)/
```

Comment:
Probably don't want to make the first alpha token optional (unless I misunderstand something).  At a minimum, we should be documenting the different kinds of tags from which we expect to build releases.